package.json

```diff
@@ -21,7 +21,9 @@
   "devDependencies": {
     "depcheck": "^0.6.4",
     "eslint": "^3.5.0",
-    "fixpack": "^2.3.1"
+    "fixpack": "^2.3.1",
+    "npm-run-all": "^3.1.0",
+    "nsp": "^2.6.1"
```

Comment:
Added `nsp` to check for vulnerable modules. Because I'm funny like that.
As always, everything is debatable and undoable.